pkg/distributor/distributor.go

```diff
@@ -166,13 +164,11 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 		quit:           make(chan struct{}),
 	}
 
-	if cfg.EnableHATracker {
-		replicas, err := newClusterTracker(cfg.HATrackerConfig)
-		if err != nil {
-			return nil, err
-		}
-		d.replicas = replicas
+	replicas, err := newClusterTracker(cfg.HATrackerConfig)
+	if err != nil {
+		return nil, err
 	}
+	d.Replicas = replicas
```

Comment:
Nit: I prefer if this goes above the building of the Distributor now.

Reply:
I'm assuming you mean move the creation of the HA tracker to above the creation of the distributor, but leave this assignment. I've done that in the latest commit.